src/app/store/Config/Config.reducer.js

```diff
@@ -12,8 +12,18 @@
 import BrowserDatabase from 'Util/BrowserDatabase';
 import { UPDATE_CONFIG } from './Config.action';
 
+export const ALWAYS_UPDATE_IN_CONFIG = ['header_logo_src', 'logo_height', 'logo_width'];
+
 export const filterStoreConfig = config => Object.entries(config).reduce(
-    (acc, [key, value]) => (value ? { ...acc, [key]: value } : acc),
+    (acc, [key, value]) => {
+        if (ALWAYS_UPDATE_IN_CONFIG.includes(key)) {
+            return value
+                ? { ...acc, [key]: value }
+                : { ...acc, [key]: undefined };
```

Comment:
This way we make sure, that logo changes immediately on FE after it was changed on BE.
Explanation:
- Before
In browser database: { logo_height : 144, header_logo_src: '124124' }
In action for reducer: { logo_width : 411 }
Reducer returns merged objects: { logo_height : 144, header_logo_src: '124124', logo_width : 411 } , which is incorrect, as we are handling no-url case, no-width case and no-height case.
- Now:
After browser gets updated, problem solves, as BrowserDatabase already got updated with { logo_width : 411 } values (BrowserDatabase is not getting merged, just gets overwritten). Now reducer returns { logo_width : 411 } + { logo_width : 411 } = { logo_width : 411 } with no additional incorrect values.